vmdk_plugin/utils/fs/fs.go

```diff
@@ -60,20 +60,23 @@ type VolumeDevSpec struct {
 // DevAttachWaitPrep create the watcher.
 func DevAttachWaitPrep(name string, devPath string) (*inotify.Watcher, bool) {
 	watcher, errWatcher := inotify.NewWatcher()
+	needToSkipInotify := false
```

Comment:
Done! reverted the proposed changed and move out the else part code to resolve the warning.